src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReportsController.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.managerecallsapi.service.ReportsService
+import uk.gov.justice.digital.hmpps.managerecallsapi.service.ReportsService.Api.GetReportResponse
+import java.time.Clock
+import java.time.OffsetDateTime
+
+@RestController
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+@PreAuthorize("hasRole('ROLE_MANAGE_RECALLS')")
+class ReportsController(
+  @Autowired private val reportsService: ReportsService,
+  @Autowired private val clock: Clock
+) {
+
+  @GetMapping("/reports/weekly-recalls-new")
+  fun weeklyRecallsNew(): GetReportResponse {
+    val now = OffsetDateTime.now(clock)
```

Comment:
feels odd to have the date here, but 🤷‍♂️

Reply:
The idea was to use the clock as early as possible and as few times as possible - as it is only present to support e.g. TZ config via Spring and testing.